styles.css

```diff
@@ -0,0 +1,392 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+/*
+
+Navigation Bar
+
+*/
+nav {
+  border: 1px black solid;
```

Comment:
Its recommended to give a class name to the tag and add styles for this class name instead of styling the tag directly since if you are using the nav tag in any other place on the application it will affect it also.